com.rtosta.zapzap.yaml

```diff
@@ -25,9 +25,10 @@ cleanup:
   - /lib/pkgconfig
   - '*.la'
   - '*.a'
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
 modules:       
-  - pyqt6.yml
-  - pyqt6-webengine.yml
+  - name: PyQtWebEngineApp
```

Comment:
```suggestion
```

This an empty module.

Reply:
I didn't understand the change. 
I followed the documentation: https://github.com/flathub/com.riverbankcomputing.PyQt.BaseApp/wiki

Reply:
And the documentation use this to indicate "your app". Not an empty module like you did. There is nothing special.

Reply:
So my change as suggested is to remove the line.

Reply:
I noticed it later when it flagged. Thanks

Reply:
I don't understand why it failed after the merge

Reply:
infra problem, I did retrigger.